image-builder.spec

```diff
@@ -63,6 +63,12 @@ install -m 0644 -vp distributions/*			%{buildroot}%{_datadir}/image-builder/dist
 %if %{with tests} || 0%{?rhel}
 install -m 0755 -vd					%{buildroot}%{_libexecdir}/tests/image-builder
 install -m 0755 -vp _bin/image-builder-tests		%{buildroot}%{_libexecdir}/tests/image-builder/
+
+%if %{with tests} || 0%{?rhel}
+install -m 0755 -vd                                             %{buildroot}%{_libexecdir}/tests/image-builder
+install -m 0755 -vp test/cases/*                                %{buildroot}%{_libexecdir}/tests/image-builder/
```

Comment:
I'm nitpicking here but is there a reason to have en entirely new if block instead of copying the last line to the block above ?

Reply:
Just trying it out on schutzbot, seems like it isn't entirely correct yet, indeed there's no reason for this and i'll change it :)

Reply:
@Gundersanne , the port I uses here is 8086, which was 8087 in schutzbot container. Not sure if this is the reason? Should we use 8086 or 8087 here?

Reply:
Currently the container is spun up at :8087, so the integration test should test against :8087

Reply:
Thanks! Changed to port 8087.

Reply:
It still failed in port 8087. I just added a port verification in the api.sh and removed silence mode in curl to see what happened here.

Reply:
Ah, it didn't start the container. Added podman run command in the api.sh.

Reply:
There's no jq in RHEL-8 and fedora-33. Install jq before test.